README.md

```diff
@@ -320,14 +320,16 @@ mev_type: null
 
 # Parameters if MEV is used
 mev_params:
-  # The image to use for MEV boot relay
+  # The image to use for MEV boost relay
   mev_relay_image: flashbots/mev-boost-relay
   # The image to use for the builder
   mev_builder_image: ethpandaops/flashbots-builder:main
   # The image to use for the CL builder
   mev_builder_cl_image: sigp/lighthouse:latest
   # The image to use for mev-boost
   mev_boost_image: flashbots/mev-boost
+  # Parameters for MEV Boost
```

Comment:
can you make it clear that this will override completely?

Reply:
I think it makes sense to replace the args, as if you use a non flashbot's builder, then you most likely will have completely different flags to define.

Reply:
yeah also i belive this an important design decision for ethereum-package. Do we want to maintain seperate module for each type of impl or a generic impl but allow to override the args.

Reply:
@barnabasbusa fair enough! :-)